### PR TITLE
Add docs on #channel_info and #channels

### PR DESCRIPTION
**Pull request by jackfranklin**
@leggetter or @hamchapman what say you?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,16 @@ Pusher['a_channel'].trigger('an_event', {:some => 'data'})
...
+
+This gem provides methods for accessing information from the [Pusher REST API](https://pusher.com/docs/rest_api). The documentation also shows an example of the responses from each of the API endpionts.
+
+The following methods are provided by the gem. If you wish to query an API endpoint that doesn't have a specific method read on to learn how to make generic request to the REST API.
+
+- `Pusher.channel_info('channel_name')` returns information about that channel.
+
+- `Pusher.channels` returns information about all the channels in your Pusher application.
+
 ### Generic requests to the Pusher REST API
```
Comment (jackfranklin) [inline README.md]: Pondering removing this bit - the only actual other API endpoint that could be requested is `GET /apps/[app_id]/channels/[channel_name]/users`. I wonder if it's worth just adding that as a method and then we can lose the "Generic requests" bit all together, which would be nice.

Comment (hamchapman) [inline README.md]: I'd be tempted to just add it explicitly

**Comment by jackfranklin** [comment]
@hamchapman added `channel_users`. When this is merged I'll cut a new release & update changelog. Given this isn't breaking we can do 0.15.2.

**Comment by leggetter** [comment]
I don't get why the gem isn't a v1.x.x Any ideas? It's probably our most mature HTTP API library.

**Comment by hamchapman** [comment]
Nice! And there's a delegator this time :dancers: 

And yeah, may as well make this a 1.0.0. It really should have been for a while

**Pull request merged**